docs/how-to-docs/cert-manager.md

```diff
@@ -211,28 +255,399 @@ spec:
+- Delete all your `Certificate` resources. You might want to back them up before doing that.
+- Wait a minute or so and verify that the certificate resources have not been re-created with `kubectl get certificates -n my-namespace`. If a certificate is re-cretated, double-check your ingress annotations.
+
+Note that:
+
+1) All `Certificate` resources should be deleted: the ones managed by the ingress shim controller and the ones for which you have `Certificate` manifests.
+2) Although the certificate resources have been deleted, this will not affect the secrets containing the TLS private key and signed certificate. That's because, by default, cert-manager does not delete the secrets specified in a certificate object when that certificate object is deleted. This means that your service will continue having a valid certificate until it expires, so you have some time (usually at least 4 weeks) to complete the migration.
+
+An example of removing the annotations can be found in [kube-example-app](https://github.com/UKHomeOffice/kube-example-app/commit/a161ea22563e45fcd87141933118d2102caa4dc1).
+
+##### Step 2 - updating the resources for v0.11+
+
+The following examples are based on the [kube-example](https://github.com/ukhomeoffice/kube-example-app) project.
+
+An example of updating annotations and labels can be found in [kube-example-app](https://github.com/UKHomeOffice/kube-example-app/commit/379a6da037a91089726bd0b335db87a3e723208e) but they are also explained below.
+
+###### External Ingress changes
+
+Changes required for websites or services exposed externally
+
+The following `Ingress` resource with v0.8 annotations:
+
 ```YAML
 apiVersion: extensions/v1beta1
-  kind: Ingress
-  metadata:
-    annotations:
-      certmanager.k8s.io/acme-challenge-type: dns01
-      certmanager.k8s.io/acme-dns01-provider: route53
-      certmanager.k8s.io/enabled: "true"
-      kubernetes.io/ingress.class: nginx-internal
-    name: example
-  spec:
-    rules:
-    - host: mysite.example.com
-      http:
-        paths:
-        - backend:
-            serviceName: service_name
-            servicePort: 443
-          path: /
-    tls:
-    - hosts:
-      - mysite.example.com
-      - example.com
-      secretName: example-tls
+kind: Ingress
+metadata:
+  name: {{ .DEPLOYMENT_NAME }}-external
+  annotations:
+    certmanager.k8s.io/acme-challenge-type: "http01"
+    certmanager.k8s.io/enabled: "true"
+    ingress.kubernetes.io/backend-protocol: "HTTPS"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: "nginx-external"
+spec:
+  rules:
+  - host: {{ .APP_HOST_EXTERNAL }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .DEPLOYMENT_NAME }}
+          servicePort: 10443
+        path: /
+  tls:
+  - hosts:
+    - {{ .APP_HOST_EXTERNAL }}
+    secretName: {{ .DEPLOYMENT_NAME }}-external-tls
+```
+
+should be changed to `Ingress` resource with the following v0.11 annotations:
```

Comment:
You mean v0.12+ annotations?

Reply:
I did mean v0.11, although it should have read v0.11+ as the annotation changes have been enforced in v0.11

Reply:
Same as above, I think it should have been v0.11+